public/app/core/components/RolePicker/RolePicker.tsx

```diff
@@ -64,7 +64,9 @@ export const RolePicker = ({
     let vertical = -offsetVertical;
 
     if (distance < MENU_MAX_HEIGHT + 20) {
-      vertical = offsetVertical;
+      // Off set to display the role picker menu at the bottom of the screen
```

Comment:
nit: could add some comment when this happens `if (distance < MENU_MAX_HEIGHT + 20) {`, is it possible to understand what this condition is in words?

Reply:
This is a great suggestion! I couldn't come up with anything more descriptive than the statement itself. If the distance is under the menu maximum height plus 20px, do that.

I'm open to suggestions 🙏